Traceback (most recent call last):
  File "/home/ubuntu/mlproject/app.py", line 7, in <module>
    with open('./model.pkl', 'rb') as model_pkl:
FileNotFoundError: [Errno 2] No such file or directory: './model.pkl'
python: can't open file '/home/ubuntu/mlproject/your_script.py': [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/home/ubuntu/mlproject/app.py", line 7, in <module>
    with open('./model.pkl', 'rb') as model_pkl:
FileNotFoundError: [Errno 2] No such file or directory: './model.pkl'
Traceback (most recent call last):
  File "/home/ubuntu/mlproject/app.py", line 7, in <module>
    with open('./model.pkl', 'rb') as model_pkl:
FileNotFoundError: [Errno 2] No such file or directory: './model.pkl'
